templates/tapestry.js

```diff
@@ -2513,7 +2510,7 @@ function tapestryTool(config){
                     node.typeData = content.typeData
                 }
 
-                if (node.mediaType !== "accordion" && tapestry.dataset.nodes[index].typeData.progress) {
```

Comment:
We should keep this condition as it has caused some issues in the past because it was not set. We can change it to `&& node.typeData.progress` I guess. We should make sure this is on master as well